cli/cmd/install.go

```diff
@@ -1261,3 +1263,33 @@ func toIdentityContext(idvals *identityWithAnchorsAndTrustDomain) *pb.IdentityCo
 		Scheme:             idvals.Identity.Issuer.Scheme,
 	}
 }
+
+// Read a resource file from a path or URL
+func readFile(path string) ([]byte, error) {
+	if isValidURL(path) {
+		resp, err := http.Get(path)
```

Comment:
I wonder whether we can extract the logic that is here https://github.com/linkerd/linkerd2/blob/main/cli/cmd/inject_util.go#L152 into something that we can reuse right here.

Reply:
That's a good idea. Sounds like we might even be able to reuse the entire `read()` function.